From 0a67abcb925d69166404df82f74c79ac6f98c3e6 Mon Sep 17 00:00:00 2001
From: Wasim Nazir <quic_wasimn@quicinc.com>
Date: Tue, 25 Feb 2025 12:08:03 +0530
Subject: [PATCH 1/2] PENDING: dt-bindings: arm: qcom: Rename qcs9075-rb8 to
 qcs9075-iq-9075-evk

Update board definition of RB8 to EVK as per new product name.

Change-Id: Ief6569d6a739a767e8a2e971886a0efd34bcc04c
Signed-off-by: Wasim Nazir <quic_wasimn@quicinc.com>
Upstream-Status: Pending
---
 Documentation/devicetree/bindings/arm/qcom.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/arm/qcom.yaml b/Documentation/devicetree/bindings/arm/qcom.yaml
index 243bc5fb3ac6..ec4568e5d2a0 100644
--- a/Documentation/devicetree/bindings/arm/qcom.yaml
+++ b/Documentation/devicetree/bindings/arm/qcom.yaml
@@ -902,7 +902,7 @@ properties:
 
       - items:
           - enum:
-              - qcom,qcs9075-rb8
+              - qcom,qcs9075-iq-9075-evk
               - qcom,qcs9075-ride
               - qcom,qcs9075-ride-r3
           - const: qcom,qcs9075
-- 
2.48.1

